### PR TITLE
Add lib directory to switch statement

### PR DESCRIPTION
**Pull request by lewiscowper**
This change should allow us to add the `directories.lib` property by default, if a directory named "lib" exists in the directory npm init is called. Or at least that's the intention of this change.

If I've not found the correct area to change, feel free to close this, ideally with a pointer to a more appropriate package. This same structure exists in promzard<sup>[0]</sup>, and I'm happy to send a PR to add this change in there if keeping these two in sync is desired. 

[0] - https://github.com/npm/promzard/blob/9d655a45d1bc99ece500c1e0dadb10d39bfec57a/example/npm-init/init-input.js#L65-L80

### CODE REVIEW

**Comment by zkat** [comment]
Hey! Thanks for the patch. Is there a particular problem this is meant to solve?

**Comment by lewiscowper** [comment]
Hi, wow, I snoozed this notification for a long time. Sorry about that.

So, I raised npm/docs#476 when I discovered that `npm init --yes` will add the `"directories": { "test": "test" }` object to the package.json when creating it. I then opened npm/npm#11644 to add documentation about the `directories.test` object.

This PR is to get `init-package.json` to the way that the docs at https://docs.npmjs.com/files/package.json#directorieslib suggest it should be, at least to my mind.

**tl;dr** — Running `npm init --yes` correctly picks up `test/`, `example/`, `doc/`, and `man/` dirs, but not a `lib/` dir, and the npm docs mention the `lib/` dir as something else we could auto-set when running `npm init --yes`.

If my code reading was successful, this change should do that.

**Comment by iarna** [comment]
Hey, thanks for this. It's been (finally!) merged and released in v1.10.0.
